app/controllers/api/internal/v2/laa_references_controller.rb

```diff
@@ -9,22 +9,21 @@ def create
           contract = NewLaaReferenceContract.new.call(**transformed_params)
           enforce_contract!(contract)
 
-          MaatLinkCreatorWorker.perform_async(
-            Current.request_id,
+          MaatLinkCreator.call(
             transformed_params[:defendant_id],
             transformed_params[:user_name],
             transformed_params[:maat_reference],
           )
```

Comment:
Should we check for the result?
that return errors, if I am not wrong as exceptions,
maybe we should rescue the exception and based on the code return different error message code?